roles/chameleon_usage/tasks/config.yml

```diff
@@ -17,3 +17,4 @@
   vars:
     mysql_host: "{{ chameleon_usage_mysql_host }}"
     root_password: "{{ database_password }}"
+    kvm_host: "{{ kvm_mysql_host }}"
```

Comment:
Without a default or something, this will fail the role at UC. Can you make it so the KVM configuration is optional and will only run if this variable (or something like `enable_usage_kvm` is True, which is how we usually toggle these things on/off.)